src/publish.ts

```diff
@@ -360,9 +299,7 @@ To publish a release you should start from a clean repo. Run "npx release-plan p
   // the end.
   const reporter = new IssueReporter();
 
-  await makeTags(solution, reporter, opts);
   await npmPublish(solution, reporter, opts, packageManager());
-  await pushTags(reporter, opts);
   await createGithubRelease(
```

Comment:
makeTags was creating 1 tag per public package in a monorepo. Now it will only create 1 tag.
I think we should either 
* continue to use makeTags and just change it to use https://octokit.github.io/rest.js/v21/#git-create-tag
or 
make `createGithubRelease` create a release+tag for each package that has changes. But that would also require more changes as the changelog in the release would need to be separated for each package